catchpoint/config.yml

```diff
@@ -0,0 +1,49 @@
+# Name of the quickstart
+name: Catchpoint- uickstart
+
+# Displayed in the UI
+title: Catchpoint-Quickstart
```

Comment:
You don't have to hyphenate the title, unless that's how you'd like the name displayed!